frontend/src/components/Inputs/Password.vue

```diff
@@ -20,18 +20,23 @@
         </p-button>
       </div>
     </div>
-    <div style="height: 30px; margin-top: -15px;" v-if="password_type == 'storepass' && !this.$store.state.validators.duplicate && !this.$store.state.app.login && !this.$store.state.validators.storepass.valid_password">             
-              <p v-if="!this.$store.state.validators.storepass.contains_eight_characters" class="validate"> 8 Characters Long, </p> 
-              <p v-if="!this.$store.state.validators.storepass.contains_number" class="validate"> Number,</p> 
-              <p v-if="!this.$store.state.validators.storepass.contains_uppercase" class="validate"> Uppercase, </p> 
-              <p v-if="!this.$store.state.validators.storepass.contains_special_character" class="validate"> Special Character </p>     
+    
+    <div style="height: 30px; margin-top: -15px;" v-if="storepass_validations">             
+      <p v-if="!this.$store.state.validators.storepass.contains_eight_characters" class="validate"> 8 Characters Long, </p> 
+      <p v-if="!this.$store.state.validators.storepass.contains_number" class="validate"> Number,</p> 
+      <p v-if="!this.$store.state.validators.storepass.contains_uppercase" class="validate"> Uppercase, </p> 
+      <p v-if="!this.$store.state.validators.storepass.contains_special_character" class="validate"> Special Character </p>     
     </div>
-    <div style="height: 30px; margin-top: -15px;" v-if="password_type == 'keypass' && !this.$store.state.validators.duplicate && !this.$store.state.app.login && !this.$store.state.validators.keypass.valid_password">             
-              <p v-if="!this.$store.state.validators.keypass.contains_eight_characters" class="validate"> 8 Characters Long, </p> 
-              <p v-if="!this.$store.state.validators.keypass.contains_number" class="validate"> Number,</p> 
-              <p v-if="!this.$store.state.validators.keypass.contains_uppercase" class="validate"> Uppercase, </p> 
-              <p v-if="!this.$store.state.validators.keypass.contains_special_character" class="validate"> Special Character </p>     
+    <div style="height: 30px; margin-top: -15px;" v-if="!storepass_validations"/> 
+
+    <div style="height: 30px; margin-top: -15px;" v-if="keypass_validations">             
+      <p v-if="!this.$store.state.validators.keypass.contains_eight_characters" class="validate"> 8 Characters Long, </p> 
+      <p v-if="!this.$store.state.validators.keypass.contains_number" class="validate"> Number,</p> 
+      <p v-if="!this.$store.state.validators.keypass.contains_uppercase" class="validate"> Uppercase, </p> 
+      <p v-if="!this.$store.state.validators.keypass.contains_special_character" class="validate"> Special Character </p>     
     </div>
```

Comment:
This exploded the margins for me. Tweaked them a bit and found that below was the sweet spot for me.

```VueJS
    <div style="height: 30px; padding-top: 20px; margin-top: -30px; margin-bottom: -10px;" v-if="storepass_validations">             
      <p v-if="!this.$store.state.validators.storepass.contains_eight_characters" class="validate"> 8 Characters Long, </p> 
      <p v-if="!this.$store.state.validators.storepass.contains_number" class="validate"> Number,</p> 
      <p v-if="!this.$store.state.validators.storepass.contains_uppercase" class="validate"> Uppercase, </p> 
      <p v-if="!this.$store.state.validators.storepass.contains_special_character" class="validate"> Special Character </p>     
    </div>
    <div style="height: 30px; margin-top: -30px; margin-bottom: -10px;" v-if="!storepass_validations"/> 

    <div style="padding-top: 20px; height: 30px; margin-top: -30px; margin-bottom: -10px;" v-if="keypass_validations">             
      <p v-if="!this.$store.state.validators.keypass.contains_eight_characters" class="validate"> 8 Characters Long, </p> 
      <p v-if="!this.$store.state.validators.keypass.contains_number" class="validate"> Number,</p> 
      <p v-if="!this.$store.state.validators.keypass.contains_uppercase" class="validate"> Uppercase, </p> 
      <p v-if="!this.$store.state.validators.keypass.contains_special_character" class="validate"> Special Character </p>     
    </div>
    <div style="height: 30px; margin-top: -30px; margin-bottom: -10px;" v-if="!keypass_validations"/>

```